.azure/templates/build.yml

```diff
@@ -79,10 +79,10 @@ jobs:
       script: |
        $MsbuildArgsDict =
        @{
-           "ProductMajor" = $env:PRODUCTMAJOR;
-           "ProductMinor" = $env:PRODUCTMINOR;
-           "BuildMajor" = $env:BUILDMAJOR;
-           "BuildMinor" = $env:BUILDMINOR;
+           "PRODUCT_MAJOR" = $env:PRODUCTMAJOR;
+           "PRODUCT_MINOR" = $env:PRODUCTMINOR;
```

Comment:
Is it necessary to change to uppercase? I saw inconsistent, e.g., SignType

Reply:
Yes, they are the build properties we defined and need to be upper-cased: https://github.com/microsoft/IIS.Common/blob/master/version/bldver.h

On the other hand, "SignType" is the property microbuild defines and follows the upper camel case.